src/less/tokens/base/tokens.less

```diff
@@ -27,6 +27,25 @@
 @color-action-disabled: @color-grey3;
 @color-action-destroy: @color-r4;
 
+// destructive
+@color-destructive-primary-text: @color-white;
+@color-destructive-primary-disabled-text: @color-white;
+@color-destructive-primary-disabled-background: @color-grey3;
+@color-destructive-secondary-text: @color-r4;
+@color-destructive-secondary-hover-text: @color-r5;
+@color-destructive-secondary-active-text: @color-r6;
+@color-destructive: @color-r4;
+@color-destructive-hover: @color-r5;
+@color-destructive-active: @color-r6;
+
+// destructive spinners
+@color-destructive-primary-spinner-background: #ec7089; // short section
```

Comment:
Should use a variable for this and define it in primitives.

Reply:
We could. I was debating that but since design did not have established token names, to avoid future confusion, I did not want to start a new convention that may need to be changed later.